lib/workers/repository/update/pr/index.ts

```diff
@@ -325,10 +328,11 @@ export async function ensurePr(
       if (GlobalConfig.get('dryRun')) {
         logger.info(`DRY-RUN: Would update PR #${existingPr.number}`);
       } else {
+        const newPrBody = updateRenovateBody(prBody, existingPr.body);
```

Comment:
can't we reuse `prBody` ?

Reply:
The purpose of this PR is to prevent RenovateBot from replacing a modified PR body with Renovate Body only.
We want to keep everything outside of `<!--renovate:start--><!--renovate:end-->` and only replace the part within the renovate tags to prevent removing custom edits in the `existingPr` description.

Using `prBody` would replace the full body (containing edits from devs/PR Templates) with the latest renovate body.